applications/challengedac.md

```diff
@@ -0,0 +1,109 @@
+
+# EOS Network Foundation Grant Proposal
+
+
+- **Project Name:** ChallengeDac
+- **Team Name:** ChallengeDac
+- **EOS Payment Address:** g4ztaojzgage
+- **[Level](https://github.com/eosnetworkfoundation/grant-framework#grant-levels):** 2
+- **Pomelo Grant(s):** [https://pomelo.io/grants/chlcontent](https://pomelo.io/grants/chlcontent)
+- **Project is Open-Source:** Yes
+- **Project was part of Token sale:** No
+- **Repository where Project resides:** [https://github.com/ChallengeDAC](https://github.com/ChallengeDAC)
+
+## Contact
+
+- **Contact Name:** Chase Eskam
+- **Contact Email:** [chaceaeskam@gmail.com](mailto:chaceaeskam@gmail.com)
+- **Website:** [https://www.challengedapp.io/](https://www.challengedapp.io/)
+
+
+## Project Overview
+
+
+
+### Overview
+
+ChallengeDac is a decentralized autonomous organization (DAO) built on the EOS and Telos blockchain that aims to provide a platform for people to create, discover, and participate in Challenges. ChallengeDAC allows users to create Challenges in a variety of categories, such as education, fitness, and entertainment, and to set the terms and conditions for participation. Other users can then discover and participate in these challenges, earning rewards for successfully completing them. ChallengeDAC uses smart contracts to enforce the rules of each challenge and to automate the distribution of rewards. CurrentlychallengeDac has a user's base of over 20,000.
+
+### Project Details
+
+In this proposal, we suggest making our Blockchain technology a public EOS Client Dart SDK to expand the use of EOS through organizations or individuals who want to utilize our technology for making the transfer, withdrawing, and tipping smart contracts. This not only benefits those who want to use these unique features but for our application stability, this open-source projects will be maintained frequently to increase stability and reliability. After that, we recommend the creation of 5,000 new, unique pieces of EOS-related content. We plan on achieving this by creating 1000-3000 new custom Challenges that will be posted on the ChallengeDac platform asking users to create their own new unique EOS-related content and rewarding 10-20 users per smart contract Challenge posted. The plan will be to Post Challenges that will reward users for creating blogs, videos, tweets, and other media to help educate the public about EOS and post the content on major social platforms like Snapchat, Facebook, Instagram, and Twitter. Afterward users can share the links of the EOS related content posts they made on the ChallengeDac Platform and be able to collect their ChallengeDac rewards. This will also help grow and accelerate the adoption of ChallengeDac which is one of the most used EOS dapp.
```

Comment:
Is there any way to measure efficiency of "challenge" marketing campaigns in bringing in convertible "attention"? It really reminds me of a bounty campaigns pitfall that tend to bring in temporary users interested only in the bounty and they switch to the next project.

Reply:
Our project and platform aim is to provide a stable place where Challenge creators and participants can cooperate and create value for one another well into the future. 
Efficiency is going to be measured and viewed differently based on every Challenge creator's goals. For ENF-sponsored Challenges we can make a game plan for the results we are seeking to achieve!